packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -25,7 +26,6 @@ import type { RoutingFormWithResponseCount } from "../../components/SingleForm";
 import SingleForm, {
   getServerSidePropsForSingleFormView as getServerSideProps,
 } from "../../components/SingleForm";
-import "../../components/react-awesome-query-builder/styles.css";
```

Comment:
why this line is removed?

Reply:
As the CSS also needs to be migrated to 6.x  according to https://github.com/ukrbublik/react-awesome-query-builder/blob/master/README.md#migration-to-600

If I'm wrong, it would be very grateful, if you suggest me the changes.

Reply:
Yes, but you should change only if relevant file from library was used 
- import 'react-awesome-query-builder/lib/css/styles.css';
+ import '@react-awesome-query-builder/ui/css/styles.css';

I am not sure though, I can also be wrong.

Reply:
Has it been added somewhere else? My concern is, it must be adding certain styles, don't we need them now?